### ISSUE TITLE
DOC: setup.py claims we use "ATLAS BLAS / LAPACK library"

### ISSUE THREAD

**Issue by mattip** [numpy/numpy#11793]
the top-level `setup.py`'s docstring has this:
```
Windows wheels are linked against the ATLAS BLAS / LAPACK library, restricted
to SSE2 instructions, so may not give optimal linear algebra performance for
your machine.
```
Aren't we using OpenBLAS on all platforms?

**Comment by rgommers** [comment]
Indeed, that statement is outdated. I see a few more things to fix in that docstring, will try to send a PR today or tomorrow.

**Comment by mattip** [comment]
I noticed it because that is what is displayed on [PyPI](https://pypi.org/project/numpy/), would be nice to fix for 1.15.1

**Comment by charris** [comment]
I don't think it is worth delaying 1.15.1 for this, as it is unlikely that anyone notices.
